index.html

```diff
@@ -39,6 +39,7 @@
       #bot.dark .accent-stroke { stroke: rgba(153, 153, 153, 0.25); }
 
       #bot-extras {
+        pointer-events: none;
```

Comment:
just makes editing in a browser a little easier, so that the overlay bit doesn't grab every click. ie, this doesn't matter to adding this stuff.